### ISSUE TITLE
Copying images to wayland apps pastes only BMP images

### ISSUE THREAD

**Issue by abhishekchak52** [microsoft/wslg#833]
### Windows build number:

22622.598

### Your Distribution version:

22.04

### Your WSL versions:

WSL version: 0.67.6.0
Kernel version: 5.15.62.1
WSLg version: 1.0.44
MSRDC version: 1.2.3401
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22622.598

### Steps to reproduce:

1. Install `wl-clipboard` using `sudo apt get wl-clipboard`.
2. Copy an image in a Windows application (e.g. image from a website).
3. In a WSL terminal, run `wl-paste -l`

### WSL logs:

[WslLogs-2022-09-21_20-59-55.zip](https://github.com/microsoft/wslg/files/9622099/WslLogs-2022-09-21_20-59-55.zip)
[pulseaudio.log](https://github.com/microsoft/wslg/files/9622101/pulseaudio.log)
[stderr.log](https://github.com/microsoft/wslg/files/9622102/stderr.log)
[weston.log](https://github.com/microsoft/wslg/files/9622103/weston.log)

### Expected behavior:

A list of other image formats is expected, such as (part of output on a computer running Ubuntu 22.04 with the same version of wl-clipboard):
```
image/bmp
image/jpeg
image/png
text/html
```

### Actual behavior:

It is only possible to paste the image in the BMP format.
```
text/html
image/bmp
```

**Comment by zcobol** [comment]
From `wl-paste` manual when using `-l` argument `Instead of pasting the selection, output the list of MIME types it is offered in`. Since WSLg is using RDP I think that's why `wl-paste` thinks there's an `image/bmp` in the clipboard. `wl-paste` is getting the info actually form `xdg-mime`.

This is from a Fedora wayland session, not WSL:

```
[zcobol@z15 ~]$ wl-paste -l
text/x-moz-url
text/html
image/png
MULTIPLE
SAVE_TARGETS
TARGETS
TIMESTAMP
```

and the actual image is a JPG:

```
zcobol@z15 ~]$ wl-paste
http://muezza.ca/thoughts/favourite_computer/Header.jpeg
A picture of a Mac Classic II
```

**Comment by abhishekchak52** [comment]
Correct. I believe that the expected behavior is that it should be identical to the native Ubuntu/Fedora experience. 

Instead when I copy an image from a website, for example, this is what I get as output to `wl-paste`:

```
<html><body>
<!--StartFragment--><img src="https://qutip.org/images/logo.png" height="125px"><!--EndFragment-->
</body>
</html>
```

Once pasted, however, the file does behave like a BMP image file. 
If indeed it is RDP that is causing this, what is a way to copy images accurately?

**Comment by hideyukn88** [comment]
Hi all, currently only image/bmp (aka CF_DIB at Window's world) is supported as image format in clipboard between Windows and Wayland.

https://github.com/microsoft/weston-mirror/blob/603601f2fe63e43f57a58cef6c880d054731dc6e/libweston/backend-rdp/rdpclip.c#L92

Thanks!
